.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,25 +11,19 @@ assignees: ''
 Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
 
 For more non-trivial issues, we would also appreciate if you included the following details:
-
-### Running environment
  - Git Machete plugin version
  - IDE type [e.g. IntelliJ, PyCharm]
  - IDE version [e.g. 2020.1.4]
  - Operating system [e.g. Windows]
 
+Note that this information will be included automatically when reporting an error directly from IDE's error dialog.
```

Comment:
This was already the case since the very important @DcortezMeleth 's PR https://github.com/VirtusLab/git-machete-intellij-plugin/pull/786 ... we've finally started getting stack traces in error reports